.env-example

```diff
@@ -1,2 +1,4 @@
 SECRET_KEY="this_IS_secr3t"
 DEVENV="docker"
+MAPIT_URL="https://mapit.mysociety.org/"
```

Comment:
Should we also add the `MAPIT_URL` setting to the `web` container’s environment in `docker-compose.yml`?